### DISCUSSION TITLE
credential problem.

### DISCUSSION THREAD

**Discussion by ChamilaShantha** [nasirkhan/laravel-starter#589] [General]
i use mysql to develop this project i need all step to install that project. after install it. i can not log to the system there is credential problem.

**Comment by nasirkhan** [comment]
can you share a screenshot? 
as mentioned in the doc, if you run the migration, the super admin user credentials will be `super@admin.com` and `secret`.

**Reply by ChamilaShantha** [reply]
thanks 
![image](https://github.com/nasirkhan/laravel-starter/assets/159158764/a4c205e0-de34-414e-b5d1-bb509fe9fd1a)

**Reply by ChamilaShantha** [reply]
I installed this several times, but this error occurred each time. and also used 3308 as MySQL port

**Reply by nasirkhan** [reply]
Please run the following command and it might be work, 

```php

php artisan cache:clear

```

**Reply by ChamilaShantha** [reply]
it also not worked. i feel it's, my fault. I can't understand what it is. Therefore, I asked about the step of installation.

**Reply by ChamilaShantha** [reply]
Thank you for your reply.

**Reply by ChamilaShantha** [reply]
after this command, php artisan migrate. i put this one php artisan db:seed now its ok.

**Reply by nasirkhan** [reply]
`php artisan migrate` and `php artisan db:seed` can be done using one single command: `php artisan migrate --seed`. 

if there are table in the database and you want to reset the entire database use this command:  `php artisan migrate:fresh --seed`

**Reply by ChamilaShantha** [reply]
thanks. if there is a problem, i will send that.
